code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -837,6 +842,18 @@
 	reagents.add_reagent(URANIUM, 3.6)
 	reagents.add_reagent(SPORTDRINK, 20)
 
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/canned_bread
+	name = "\improper canned bread"
+	desc = "Wow, they have it!"
+	icon_state = "cannedbread"
+	//no actual chemicals in the can
+	
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/canned_bread/pop_open(var/mob/user)
+	. = ..()
+	spawn(0.5 SECONDS)
+		playsound(src, pick('sound/effects/splat_pie1.ogg','sound/effects/splat_pie2.ogg'), 50)
+		new /obj/item/weapon/reagent_containers/food/snacks/sliceable/bread(get_turf(src))
```

Comment:
on the floor? gross

put_in_hands please

Reply:
What if you open the can with your hands full.

Reply:
put_in_hands accounts for that and drops it if your hands are full, so it's no worse